DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: snomedizer
 Type: Package
-Title: R Interface to the SNOMED CT Terminology Server REST API
+Title: R Interface to the SNOMED-CT Terminology Server REST API
```

Comment:
Small observation, in other places (e.g. rest-api.R) SNOMED-CT is written without hyphen